Mathlib/Probability/Distributions/Gaussian.lean

```diff
@@ -426,14 +451,139 @@ theorem mgf_gaussianReal (hX : p.map X = gaussianReal μ v) (t : ℝ) :
   rw [← mgf_id_map hX_meas, ← complexMGF_ofReal, hX, complexMGF_id_gaussianReal, mul_comm μ]
   norm_cast
 
+theorem mgf_fun_id_gaussianReal :
+    mgf (fun x ↦ x) (gaussianReal μ v) = fun t ↦ rexp (μ * t + v * t ^ 2 / 2) := by
+  ext t
+  rw [mgf_gaussianReal]
+  simp
+
+theorem mgf_id_gaussianReal : mgf id (gaussianReal μ v) = fun t ↦ rexp (μ * t + v * t ^ 2 / 2) :=
+  mgf_fun_id_gaussianReal
+
 /-- The cumulant generating function of a random variable with Gaussian distribution
 with mean `μ` and variance `v` is given by `t ↦ μ * t + v * t ^ 2 / 2`. -/
 theorem cgf_gaussianReal (hX : p.map X = gaussianReal μ v) (t : ℝ) :
     cgf X p t = μ * t + v * t ^ 2 / 2 := by
   rw [cgf, mgf_gaussianReal hX t, Real.log_exp]
 
+lemma integrable_exp_mul_gaussianReal (t : ℝ) :
+    Integrable (fun x ↦ rexp (t * x)) (gaussianReal μ v) := by
+  rw [← mgf_pos_iff, mgf_gaussianReal (μ := μ) (v := v) (by simp)]
+  exact Real.exp_pos _
+
+@[simp]
+lemma integrableExpSet_id_gaussianReal : integrableExpSet id (gaussianReal μ v) = Set.univ := by
+  ext
+  simpa [integrableExpSet] using integrable_exp_mul_gaussianReal _
+
+@[simp]
+lemma integrableExpSet_fun_id_gaussianReal :
+    integrableExpSet (fun x ↦ x) (gaussianReal μ v) = Set.univ :=
+  integrableExpSet_id_gaussianReal
+
 end CharacteristicFunction
 
+section Moments
+
+variable {μ : ℝ} {v : ℝ≥0}
+
+/-- The mean of a real Gaussian distribution `gaussianReal μ v` is its mean parameter `μ`. -/
+@[simp]
+lemma integral_fun_id_gaussianReal : ∫ x, x ∂gaussianReal μ v = μ := by
```

Comment:
```suggestion
lemma integral_id_gaussianReal : ∫ x, x ∂gaussianReal μ v = μ := by
```
Here there is no fun in the statement, and I don't see what another version with `id` could be, so I'd drop the `fun` from the statement.